package.json

```diff
@@ -26,7 +24,11 @@
     "sass": "sass --watch src/styles/scss:src/styles/css",
     "precommit": "lint-staged",
     "analyze": "source-map-explorer build/static/js/main.*",
-    "storybook": "start-storybook -p 9001 -c .storybook"
+    "storybook": "start-storybook -p 9001 -c .storybook",
+    "lingui-init": "lingui init",
+    "lingui-add-locale": "lingui add-locale en fr",
```

Comment:
Please remove this task, in the future we will use `lingui add-locale es jp` or any other locale we want, for now you have already installed en and fr, no need for others to run this task again.